apps/vr-tests-react-components/src/stories/InfoButton.stories.tsx

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+import { InfoButton } from '@fluentui/react-infobutton';
+import { storiesOf } from '@storybook/react';
+import { PopoverProps } from '@fluentui/react-popover';
+
+const infoButtonContent = 'This is the content of an InfoButton.';
+
+storiesOf('InfoButton', module)
+  .addStory(
+    'size',
+    () => (
+      <div style={{ display: 'flex', flexDirection: 'column', padding: '60px', gap: '80px', alignItems: 'start' }}>
+        <InfoButton id="show-content-small" size="small" content={infoButtonContent} />
+        <InfoButton id="show-content-medium" size="medium" content={infoButtonContent} />
+        <InfoButton id="show-content-large" size="large" content={infoButtonContent} />
+      </div>
+    ),
+    {
+      includeHighContrast: true,
+      includeDarkMode: true,
+    },
+  )
+  .addStory(
+    'size-open',
+    () => (
+      <div style={{ display: 'flex', flexDirection: 'column', padding: '60px', gap: '80px', alignItems: 'start' }}>
+        <InfoButton
+          id="show-content-small"
+          size="small"
+          content={{ children: infoButtonContent }}
```

Comment:
`content` can be set to a string right?
```suggestion
          content={infoButtonContent}
``` 

(super-nit) And it's probably fine to just write out the string instead of using a const.

```suggestion
          content="This is the content of an InfoButton."
```